lib/puma/cli.rb

```diff
@@ -100,6 +100,8 @@ def configure_control_url(command_line_arg)
     def setup_options
       @conf = Configuration.new do |user_config, file_config|
         @parser = OptionParser.new do |o|
+          o.set_summary_width 38
```

Comment:
Probably best to remove this in this PR.

Reply:
It did cause the output to look odd which was the reason for adding, will remove after the main discussion after the PR in general.